frontend/public/components/conditions.tsx

```diff
@@ -2,9 +2,17 @@ import * as React from 'react';
 import { useTranslation } from 'react-i18next';
 import { LinkifyExternal, Timestamp } from './utils';
 import { CamelCaseWrap } from './utils/camel-case-wrap';
-import { K8sResourceCondition } from '../module/k8s';
+import { ClusterServiceVersionCondition, K8sResourceCondition } from '../module/k8s';
 
-export const Conditions: React.FC<ConditionsProps> = ({ conditions }) => {
+export enum ConditionTypes {
+  ClusterServiceVersion = 'ClusterServiceVersion',
+  K8sResource = 'K8sResource',
```

Comment:
Would be good to add comment/explanation about why we have only these two condition types

Reply:
@jhadvig done